ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -60,6 +60,12 @@ EOF
     )
 fi
 
+echo "Setting label security.openshift.io/scc.podSecurityLabelSync value to true on the namespace \"$OO_INSTALL_NAMESPACE\""
+
+if [[ "${OO_INSTALL_NAMESPACE}" =~ ^openshift- ]]; then
+    oc label --overwrite ns openshift-logging security.openshift.io/scc.podSecurityLabelSync=true
```

Comment:
This should be `$OO_INSTALL_NAMESPACE` instead of `openshift-logging` , right?

Reply:
yes, that right. Good catch